spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -90,7 +90,7 @@ spring:
             # Metrics
 
             - GET    /metrics/**                     => hasRole('ROLE_VIEW')
-            - DELETE /metrics/**                     => hasRole('ROLE_CREATE')
```

Comment:
So, `ROLE_CREATE` is removed? Won't that be a breaking change?

Reply:
I didn't pay attention to the diff correctly.  The `/delete` operation is mapped to `ROLE_DESTROY` instead of `ROLE_CREATE`, and that correct!